### ISSUE TITLE
"Failed to execute Skype backend request GetThreadS2SRequest." when trying to get team by id

### ISSUE THREAD

**Issue by s-KaiNet** [microsoftgraph/microsoft-graph-docs#7793]
When trying to get team details in MS Graph Explorer I recieve: 

```
{
    "error": {
        "code": "BadRequest",
        "message": "Failed to execute Skype backend request GetThreadS2SRequest.",
        "innerError": {
            "request-id": "3acbc391-32d0-48eb-8758-b3b992759c6f",
            "date": "2020-04-17T12:49:05"
        }
    }
}
```
Request: 
> https://graph.microsoft.com/v1.0/teams/{guid}

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 533cbe72-3c95-04b8-5db4-cdd32d088d57
* Version Independent ID: fb69519b-3e4a-2279-cf1d-8e9c082fafd6
* Content: [Get team - Microsoft Graph v1.0](https://docs.microsoft.com/en-us/graph/api/team-get?view=graph-rest-1.0)
* Content Source: [api-reference/v1.0/api/team-get.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/master/api-reference/v1.0/api/team-get.md)
* Product: **microsoft-teams**
* Technology: **microsoft-graph**
* GitHub Login: @nkramer
* Microsoft Alias: **MSGraphDocsVteam**

**Comment by uday-teja** [comment]
Hi @nkramer even we stuck at the same issue. our production is halted because of the error. Hoping for some quick response. Thanks

**Comment by janardhan96** [comment]
Hello @uday-teja, Did you got an issue while getting channels?

**Comment by uday-teja** [comment]
Hi @janardhan96 while fetching the channels and also when i try to create team i got the same response.

**Comment by s-KaiNet** [comment]
The same for me - an error when trying to get a team and when I try to get channels.

**Comment by ashoksubedi1** [comment]
hi there, do you have any idea how i can resolve this issue, my app creating groups and all but right when it is time to create a team I get an error
https://graph.microsoft.com/v1.0/groups/0*******b7a7-0a931544db69/team PUT request

{
"error": {
"code": "BadGateway",
"message": "Failed to execute backend request.",
"innerError": {
"request-id": "2ab31a91-1093-4b2b-a570-9eb23b36519f",
"date": "2020-04-17T20:23:48"
}
}
}

**Comment by ashoksubedi1** [comment]
This issue is with any API endpoint that has the word "team" in it. Does not matter listing channels, creating teams or others. Please retest this and fix the issue. My Prod app depends on it

**Comment by uday-teja** [comment]
Hi all now everything works fine :)

**Comment by s-KaiNet** [comment]
Works for me as well.

**Comment by janardhan96** [comment]
Again, It's not working from today morining onwards.

**Comment by ashoksubedi1** [comment]
as of right now it is working for me. Thank you

**Comment by chefinDan** [comment]
I am receiving this error when making a list channels request to the Graph API beta endpoint. I am logged in as a guest user in my tenant. The 'joinedTeams' endpoint works, but I am unable to list the channels in the team that the guest user is a member of.

**Comment by sethington** [comment]
@chefinDan did you ever have any luck with solving this? we're hitting the same issue on the `/teams/{team-id}/channels` endpoint.

**Comment by coatsy** [comment]
Also seeing this problem today on `GET beta/teams/{id}` and `GET beta/teams/{id}/channels`

**Comment by robinmalik** [comment]
As I commented on this [SO ](https://stackoverflow.com/questions/59887736/problem-with-create-team-with-microsoft-graph-api) post, I'm seeing very similar to others here with `Failed to execute backend request` when trying to send a PUT request to a `group/{id}/team` (beta) endpoint to add Teams functionality to a Group.

The other method, creating the Team via the Teams endpoint is missing functionality; you can't populate members (only owners) and the endpoint returns no information on what was created - i.e. there is no Id of the object returned so you can't work with it without searching.

**Comment by robinmalik** [comment]
Today I shifted away from using the Graph API natively because I didn't want to deal with having to batch up 'members' when exceeding 20 every time I wanted to create a group, and then upgrade to a team.

I created a test of 200 unified groups using New-UnifiedGroup (haven't hit a limit when specifying members), then a few hours later tried to upgrade these to Teams using the New-Team cmdlet from the [MicrosoftTeams](https://www.powershellgallery.com/packages/MicrosoftTeams/1.1.4) module. 

Again, getting consistent `Failed to execute backend request.` errors. Despite that, it seems as though the Teams are created. Are they successfully created? Who knows. 

This is really painful to work with.
